fit/core-reference/src/main/java/org/apache/syncope/fit/core/reference/TestIdentityRecertification.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2016 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.syncope.fit.core.reference;
+
+import java.util.Date;
+import org.apache.syncope.core.persistence.api.dao.ConfDAO;
+import org.apache.syncope.core.persistence.api.dao.UserDAO;
+import org.apache.syncope.core.persistence.api.entity.conf.CPlainAttr;
+import org.apache.syncope.core.persistence.api.entity.user.UPlainAttr;
+import org.apache.syncope.core.persistence.api.entity.user.UPlainAttrUniqueValue;
+import org.apache.syncope.core.persistence.api.entity.user.User;
+import org.apache.syncope.core.provisioning.java.job.AbstractSchedTaskJobDelegate;
+import org.apache.syncope.core.workflow.activiti.task.Recertify;
+import org.apache.syncope.core.workflow.api.UserWorkflowAdapter;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class TestIdentityRecertification extends AbstractSchedTaskJobDelegate {

```

Comment:
You could always add it to both `MasterContent.xml`, but without schedule, so that it is not run automatically.
WDYT?

Finally, extending `AbstractSchedTaskJobDelegate` and putting all logic into `doExecute` is considered best-practice, as other things are performed as well (conditionally saving execution, notification and audit, ..).